catlas/parity/parity_utils.py

```diff
@@ -375,35 +270,52 @@ def get_parity_upfront(config, run_id):
         ## Iterate over steps
         for step in config["adslab_prediction_steps"]:
             ### Load the data
-            npz_path = get_npz_path(step["checkpoint_path"])
-            if os.path.exists(npz_path):
-                df = data_preprocessing(npz_path, "parity/df_pkls/OC_20_val_data.pkl")
+            model_id = get_model_id(step["checkpoint_path"])
+            if os.path.exists("catlas/parity/df_pkls/" + model_id + ".pkl"):
```

Comment:
Want to implement the file path thing?

Reply:
maybe lol

Reply:
I think this should probably be separate since its pretty unrelated